lib/b2b/discovery.ts

```diff
@@ -19,7 +19,7 @@ export interface OrganizationsResponse extends BaseResponse {
 
 export interface DiscoveryOrganizationCreateRequest
   extends CreateOrganizationRequest {
-  intermediate_session_token: string;
+  discovery_magic_links_token: string;
```

Comment:
This should stay `intermediate_session_token` right? The change should be on the discovery authenticate endpoint

Reply:
https://github.com/stytchauth/stytch-node/pull/204/commits/223b9b0c7d10327e81c45cfa8c1a7d7707972e03